test/blackbox-tests/test-cases/dune-init.t/run.t

```diff
@@ -255,31 +255,12 @@ Will not create components with invalid names
         Library names must be non-empty and composed only of the
         following
         characters: 'A'..'Z', 'a'..'z', '_' or '0'..'9'.
-  Usage: dune init [OPTION]… COMPONENT NAME [PATH]
-  Try 'dune init --help' or 'dune --help' for more information.
+  Usage: dune init library [OPTION]… NAME [PATH]
+  Try 'dune init library --help' or 'dune --help' for more information.
   [1]
   $ test -f ./_test_lib
   [1]
 
-Will fail and inform user when invalid component command is given
```

Comment:
What about these test cases? Even if it's covered by cmdliner now, these kinds of errors were useful when upgrading cmdliner.

Reply:
Ah yeah, I'd assumed this was now a cmdliner thing, and that we shouldn't be testing it in our tests here since it's not part of the init logic anymore.

You'd prefer we keep it here?